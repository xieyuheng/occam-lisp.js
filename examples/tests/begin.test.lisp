(begin
  1
  2
  3)

(begin
  (= x 1)
  x)
