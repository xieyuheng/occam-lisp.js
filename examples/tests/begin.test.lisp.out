[load] I find undefined name: y
  defining: assert-one
  to: : (lambda (x) (begin (= y 1) (assert (equal? y 1))))

